HISTORY.md

```diff
@@ -4,6 +4,16 @@ In the beginning, programmers created the databases. Now the databases were form
 
 And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.
 
+## Snowfakery 2.5
```

Comment:
Is there a reason you're leaving off the .0 here?

Reply:
Seems to be the pattern I've been using all along.